src/main/groovy/wooga/gradle/paket/unity/tasks/PaketUnityInstall.groovy

```diff
@@ -133,21 +142,17 @@ class PaketUnityInstall extends ConventionTask {
         fileTree.exclude("**/*.pdb")
         fileTree.exclude("**/Meta")
 
-        if (!getIncludeAssemblyDefinitions()) {
+        def excludeAsmDefs = !getIncludeAssemblyDefinitions()
+        if (excludeAsmDefs) {
             fileTree.exclude("**/*.${assemblyDefinitionFileExtension}")
         }
 
         def files = fileTree.files
         return files
     }
 
-    PaketUnityInstall() {
-        description = 'Copy paket dependencies into unity projects'
-        group = PaketUnityPlugin.GROUP
-    }
-
     @TaskAction
-    protected performCopy(IncrementalTaskInputs inputs) {
+    protected execute(IncrementalTaskInputs inputs) {
```

Comment:
API change ... And we can name the action how ever we want. There is no need to use the same name.

Reply:
I didn't think it would be an API change since it's a protected method invoked by gradle internally, no?

Reply:
I can extend a class and still access the protected methods. If you change the name and the visibilty this break. On the unity plugin we do this sometimes. As long as a method is not `private` it is a breaking change.

Reply:
I understand,